### PR TITLE
Feature/display all exercises

### PR DESCRIPTION
**Pull request by Witex15**

### CODE REVIEW

File: src/core/Router.php
Comment (R0kkxSynetique) [inline src/core/Router.php:1]: As described for the Route class, this class should have the constructor building the routes and adding them to the router class list.

**Review by R0kkxSynetique — APPROVED** [review]
The folder "Controllers" has a majuscule but all the others don't.
You may have to check for the PSR-12 formatting because some files are not following this nomenclature.
You may want to have a class that execute the SQL query instead of calling the "query" method  on dbConnect.

File: document/classDiag.drawio
[the remark this answers is not in the document]
Comment (Witex15) [reply document/classDiag.drawio:1]: yup removed

File: src/core/Router.php
Comment (LeZderich) [reply src/core/Router.php:1]: Done in the feature/display_home_page.

File: src/core/Route.php
[the remark this answers is not in the document]
Comment (LeZderich) [reply src/core/Route.php:1]: Done in the feature/display_home_page

File: public/index.php
[the remark this answers is not in the document]
Comment (LeZderich) [reply public/index.php:1]: Done in the feature/display_home_page

**Pull request merged**